modules/node/src/admin/admin.provider.ts

```diff
@@ -6,6 +6,7 @@ import { AdminMessagingProviderId, MessagingProviderId } from "../constants";
 import { AbstractMessagingProvider } from "../util";
 
 import { AdminService } from "./admin.service";
+import { StateChannelJSON } from "@connext/cf-core";
```

Comment:
If this is a type that's used by the world outside cf-core then we should consider moving this into cf-types or types and importing it into cf-core instead.